server/shared/src/main/scala/org/http4s/server/Server.scala

```diff
@@ -23,7 +23,7 @@ import org.log4s.getLogger
 abstract class Server {
   private[server] val logger = getLogger
 
-  def address: ip4s.SocketAddress[ip4s.Host]
+  def address: ip4s.SocketAddress[ip4s.IpAddress]
```

Comment:
Do we track anywhere that kind of changes (some migration guide for the 1.0)?

Reply:
The change from `java.net.InetSocketAddress` to `com.comast.ip4s.SocketAddress` is noted here:
https://github.com/http4s/http4s/blob/main/website/src/hugo/content/changelog.md#v100-m24-2020-08-07